lib/config/options/index.ts

```diff
@@ -618,48 +620,49 @@ const options: RenovateOptions[] = [
   },
   {
     name: 'username',
-    description: 'Username for authentication. Currently Bitbucket only.',
+    description: 'Username for authentication. Currently Bitbucket only.', // Shouldn't we use the `supportedPlatforms` property for the Bitbucket message?
```

Comment:
Yes, but do we expose that field into our docs yet?

Reply:
It's exposed for some config options at least. Check out the docs for [`azureAutoApprove`](https://docs.renovatebot.com/configuration-options/#azureautoapprove):

![azureAutoapprove-exposes-supportedPlatforms](https://user-images.githubusercontent.com/34918129/164426880-012226e6-9e79-4bdc-a8db-a0c2a0e92227.png)

---

## Do I need to change the docs generating code?

Do I need to change something in the docs generating code so it exposes `supportedPlatforms` for the `username` option?

Reply:
Seems like it's exposed so we can drop the extra description as you suggested